content/resources.json

```diff
@@ -18,7 +18,7 @@
   ],
   "categories": [
     {
-      "title": "Pilot 004",
+      "title": "Data-to-Compute",
```

Comment:
agreed. Was already merged with the previous PR https://github.com/deltaDAO/mvg-hackathon-overview/pull/4